cogs/utils/rsstools.py

```diff
@@ -0,0 +1,75 @@
+""" A collection of functions for RSS feed parsing"""
+
+import discord
+from dateutil import parser
+import regex as re
+from __main__ import botdata
+import aiohttp
+import asyncio
+from bs4 import BeautifulSoup
+
+async def get_html(url):
+    """ Takes url, returns html. No error handling"""
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url) as response:
+            html = await response.text()
+            return html
+
+
+def has_new_dota(feed):
+    """ Takes a feedparser feed for the dota 2 blog, checks to see if newest publish date is newer
+    than what is on record. This requires looking at the botdata json field "blog_date"
+    If it cannot find the field, will automatically return false
+    returns boolean"""
+    new = ""
+    old = botdata.blog_date
```

Comment:
Since nothing was added to botdata.py in this pull request, `botdata.blog_date` will not persist after a restart. I recently implemented support for additional fields persisting in botdata, this should be simple to do, you just need to add an entry in the "defaults" dict so that it looks like this:
```py
self.defaults = OrderedDict([ 
	("userinfo" , []), 
	("guildinfo" , []), 
	("dotapatch", None),
	("blog_date", "")
])
```
After that's done, it should work. Though you'll need to replace the `botdata.blog_date` with `botdata["blog_date"]` because i couldnt get the other way working the other day.